core/tests/protractor_utils/general.js

```diff
@@ -72,7 +72,8 @@ var EDITOR_URL_SLICE = '/create/';
 var PLAYER_URL_SLICE = '/explore/';
 var LOGIN_URL_SUFFIX = '/_ah/login';
 var ADMIN_URL_SUFFIX = '/admin';
-var SCRIPTS_URL_SLICE = '/scripts/';
+// TODO(gvishal): update for prod as well.
```

Comment:
mm, good question. We are actually considering removing that because in practice travis will already have done that. So, I think it's ok to run on dev -- as long as travis checks pass, things are fine. We'll deal with prod later.